src/main/java/org/prebid/server/analytics/reporter/greenbids/model/CommonMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import org.prebid.server.auction.model.AuctionContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommonMessage {
+
+    public String version;
+    public String auctionId;
+    public String referrer;
+    public double sampling;
+    public String prebid;
+    public String greenbidsId;
+    public String pbuid;
+    public String billingId;
+    public List<AdUnit> adUnits;
+    public Long auctionElapsed;
+
+    public CommonMessage(
```

Comment:
I suggest using the Builder, and build the message inside the reporter

Reply:
fixed